feat: FIT-725: Implement Command Palette

ricardoantoniocm opened the pull request:
Replaces Organization and Account icons.

Fixes some icons.

Adds OpenBook icon to represent projects, and Changelog, Blog and Support icons.

<img width="247" height="210" alt="image" src="https://github.com/user-attachments/assets/275b605a-dede-4575-861a-8c5e68b6d8e6" />
<img width="231" height="112" alt="image" src="https://github.com/user-attachments/assets/21f5c710-0e84-45dd-a80c-53e0dce4fd3e" />
<img width="94" height="90" alt="image" src="https://github.com/user-attachments/assets/0fd6073e-16b3-444a-848f-c158f58e0b2d" />



<!--

This description MUST be filled out for a PR to receive a review. Its primary purposes are:

 - to enable your reviewer to review your code easily, and
 - to convince your reviewer that your code works as intended.

Some pointers to think about when filling out your PR description:
 - Reason for change: Description of problem and solution
 - Screenshots: All visible changes should include screenshots.
 - Rollout strategy: How will this code be rolled out? Feature flags / env var / other
 - Testing: Description of how this is being verified
 - Risks: Are there any known risks associated with this change, eg to security or performance?
 - Reviewer notes: Any info to help reviewers approve the PR
 - General notes: Any info to help onlookers understand the code, or callouts to significant portions.

You may use AI tools such as Copilot Actions to assist with writing your PR description (see https://docs.github.com/en/copilot/using-github-copilot/using-github-copilot-for-pull-requests/creating-a-pull-request-summary-with-github-copilot); however, an AI summary isn't enough by itself. You'll need to provide your reviewer with strong evidence that your code works as intended, which requires actually running the code and showing that it works.

-->

ricardoantoniocm committed:
feat: FIT-725: Implement Command Palette

ricardoantoniocm committed:
Improves book icon. Adds Support icon.

ricardoantoniocm committed:
Fixes Support icon.

ricardoantoniocm committed:
Fixes color of Book icon.

ricardoantoniocm committed:
Removes unnecessary opacity from Book icon.

ricardoantoniocm committed:
Fixes door icon.

ricardoantoniocm committed:
Fixes Terminal icon.

ricardoantoniocm committed:
Improves Organization icon. Adds Account & Settings, Blog, Changelog icons.

ricardoantoniocm commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17993246019)
> Successfully merged:  create mode 100644 web/libs/ui/src/assets/icons/user-edit.svg

ricardoantoniocm committed:
Merge branch 'develop' into 'fb-fit-725/implement-command-palette'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17993246019

ricardoantoniocm committed:
Adds bottom-right alignment for tooltip.

ricardoantoniocm committed:
Improves workspaces icon. Introduces project icon.

yyassi-heartex commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/18041731396)
> Successfully merged:  create mode 100644 label_studio/tests/sdk/test_project_annotators_api.py

yyassi-heartex committed:
Merge branch 'develop' into 'fb-fit-725/implement-command-palette'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18041731396

borisheartex commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/18088101631)

borisheartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18088101631

borisheartex committed:
Merge branch 'develop' into 'fb-fit-725/implement-command-palette'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18088101631

yyassi-heartex committed:
Merge branch 'develop' into fb-fit-725/implement-command-palette

yyassi-heartex committed:
Merge remote-tracking branch 'origin/develop' into fb-fit-725/implement-command-palette

ricardoantoniocm commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/18133161528)
> Successfully merged:  create mode 100644 .github/workflows/clean-github-cache.yml

ricardoantoniocm committed:
Merge branch 'develop' into 'fb-fit-725/implement-command-palette'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18133161528

ricardoantoniocm commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/18161572450)
> Successfully merged:  create mode 100644 web/libs/core/src/providers/AuthProvider.tsx

ricardoantoniocm committed:
Merge branch 'develop' into 'fb-fit-725/implement-command-palette'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18161572450

ricardoantoniocm commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/18200335110)
> Successfully merged:  10 files changed, 107 insertions(+), 26 deletions(-)

ricardoantoniocm committed:
Merge branch 'develop' into 'fb-fit-725/implement-command-palette'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18200335110

ricardoantoniocm commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/18202774764)
> Successfully merged: Already up to date.

ricardoantoniocm commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/18203116959)

ricardoantoniocm committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18203116959

yyassi-heartex committed:
Merge branch 'develop' into fb-fit-725/implement-command-palette

yyassi-heartex committed:
forcing an update

yyassi-heartex committed:
linting
